lib/index.js

```diff
@@ -0,0 +1,32 @@
+const GitHub = require("gh.js");
+
+/**
+ * sortByStars
+ * Sorts the user repos by stars.
+ *
+ * @name sortByStars
+ * @function
+ * @param {String} username The GitHub username.
+ * @param {String} token An optional access token.
+ * @param {Function} callback The callback function.
+ * @return {gh.js} The `gh.js` instance.
+ */
+function sortByStars(username, token, callback) {
+    if (typeof token === "function") {
+        callback = token;
+        token = undefined;
+    }
+
+    var gh = new GitHub({ token: token });
+    gh.get(`users/${username}/repos`, { all: true }, (err, repos) => {
```

Comment:
@alysonla Not sure if you know this, but we're living in a world where we can use string templates like this. :smile: 

Instead of `"users/" + username + "/repos"` we do ```` `users/${username}/repos` ````. :sparkle: